thrimshim/thrimshim/main.py

```diff
@@ -681,7 +681,7 @@ def add_template(artist=None):
 	with app.db_manager.get_conn() as conn:
 		#check if name is already in the database
 		query = sql.SQL("""
-			SELECT name FROM events WHERE name = %s 
+			SELECT name FROM templates WHERE name = %s 
```

Comment:
This same error is in `thrimshim.update_template` in a couple places

Reply:
Added those to the same commit.